### DISCUSSION TITLE
blog/2023-03-30-causal-inference-with-ordinal-regression/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#18] [Comments]
# blog/2023-03-30-causal-inference-with-ordinal-regression/

Part 7 of the GLM and causal inference series.

https://timely-flan-2986f4.netlify.app/blog/2023-03-30-causal-inference-with-ordinal-regression/

**Comment by sjwild** [comment]
Overall, good post. TEs with ordinal or categorical outcomes suck because there is no easy way to summarize it. [AGRESTI AND Tarantola](https://onlinelibrary.wiley.com/doi/abs/10.1111/stan.12130) cover some ways to talk about effects for ordinal models.

There are a couple other ATEs you can estimate, including the probability that $\mathbb{E}(y_1) > \mathbb{E}(y_0)$. See [Lu et al 2018](https://journals.sagepub.com/doi/10.3102/1076998618776435).

In another paper, [Lu et al](https://onlinelibrary.wiley.com/doi/abs/10.1111/biom.13148) agree with [Agresti and Kateri](https://onlinelibrary.wiley.com/doi/abs/10.1111/biom.12565) that relative effects might be better than ATEs for ordinal models. This might be something you want to explore here. 

I personally prefer to use the difference in probabilities for each category, but I completely understand wanting to report the probability that $\mathbb{E}(y_1) > \mathbb{E}(y_0)$.

It's outside the scope of your post, but you might be interested in a recent preprint by [Stoetzer et al](https://osf.io/preprints/g6skh/), who look st causal inference with a latent variable and IRT.

**Reply by ASKurz** [reply]
Thanks for the links! I'll check them out and, no doubt, add refer to them in the post.
